topics/sequence-analysis/tutorials/annotation-with-prokka/tours/tour.yaml

```diff
@@ -1 +1,195 @@
+    element: '#tool-search'
+    content: Click on the "JBrowse" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjbrowse%2Fjbrowse%2F0.7.0.3"]
+        .tool-old-link
+  - title: Visualize the annotation
+    element: '#tool-search'
+    content: |-
+      Execute the tool with parameters:
+      <ul>
+        <li>“Reference genome to display” to `Use a genome from history`</li>
+        <li>Select the reference genome” to `Prokka on data XX.fna`</li>
+        <li>“Produce Standalone Instance” to `Yes`</li>
+        <li>“Genetic Code” to `11: The Bacterial, Archaeal and Plant Plastid Code`</li>
+        <li>“JBrowse-in-Galaxy Action” to `New JBrowse Instance`</li>
+        <li>Click on "Insert track group"<ul>
+          <li>“Track Type” to `GFF/GFF3/BED/GBK Features`</li>
+          <li>“GFF/GFF3/BED Track Data” to `Prokka on data XX:gff`</li>
+          <li>“Track Visibility” to `On for new users`</li>
+          <li>“JBrowse Track Type [Advanced]” to `Canvas Features`</li>
+          <li>Click on “JBrowse Styling Options [Advanced]”</li>
+          <li>“JBrowse style.label” to `product,name,id`</li>
+          <li>“Track Visibility” to `On for new users`</li></ul>
+        </li>
+        <li>Click "Execute" button below</li>
+      </ul>
+    position: left
+  - title: View annotated features in JBrowse
+    content: >-
+      A new file will be created in your history, this contains the JBrowse
+      interactive visualisation. We will now view its contents and play with it
+    backdrop: true
+  - title: View annotated features in JBrowse
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      Inspect the JBrowse on data XX and data XX - Complete file by clicking on
+      the eye icon. The JBrowse window will appear in the centre Galaxy panel.
+    position: left
+  - title: View annotated features in JBrowse
+    content: |-
+      Display all the tracks and practice maneuvering around<ul>
+        <li>Click on the tick boxes on the left to display the tracks</li>
+        <li>Select contig 1 in the drop down box. You can only see one contig displayed at a time.</li>
+        <li>Zoom out by clicking on the `minus` button to see sequence reads and their coverage (the grey graph)</li>
+        <li>Zoom in by clicking on the `plus` button to see annotations.</li>
+        <li>JBrowse displays the sequence and a 6-frame amino acid translation.</li>
+        <li>Right click on a gene/feature annotation (the bars on the annotation track), then select View Details to see more information.<ul>
+          <li>gene name</li>
+          <li>product name</li>
+          <li>you can download the FASTA sequence by clicking on the disk icon. </li></ul>
+        </li>
+    backdrop: false
+  - title: Key points
+    content: |-
+      <ul>
+        <li>Prokka is a useful tool to annotate a bacterial genome.</li>
```

Comment:
Suggest also mentioning that to get a larger view, to collapse the history panel and tool panel by clicking on the arrow symbols in each panel corner.